src/Strategy/Python/PDM/Pdm.hs

```diff
@@ -137,7 +137,7 @@ reqDepType :: Req -> DepType
 reqDepType (NameReq{}) = PipType
 reqDepType (UrlReq _ _ url _) =
   if "file://" `isPrefixOf` URI.render url
-    then PathType
+    then UnresolvedPathType
```

Comment:
Wait, is this all we have to do to turn on path dep support for Python PDM projects? That's so nice!

Reply:
Also, if it is we should probably have something about PDM projects in the test plan